packages/@okta/i18n/src/properties/login.properties

```diff
@@ -651,6 +651,12 @@ registration.default.callbackhook.error=We could not process your registration a
 registration.error.userName.invalidEmail=Invalid email address
 registration.error.password.passwordRequirementsNotMet=Password requirements were not met
 registration.error.userName.notUniqueWithinOrg=An account with that email already exists
+registration.error.notUniqueWithinOrg.Email = A user with this 'Email' already exists
```

Comment:
Are Email and custom attributes the only 2 types of known errors we can run into ?

Reply:
Yes, when testing out Profile enrollment form fields, I found that 'Email' is the only pre-filled label.